README.md

```diff
@@ -50,77 +50,8 @@ See the [open issues](https://github.com/IraSoro/peri/issues) and [project](http
 
 ### How to build
 
-Install dependencies:
+You can find full instruction about the project building in [BUILDING.md](https://github.com/IraSoro/peri/blob/master/BUILDING.md) file.

```

Comment:
`You can find the full instruction about the project building in`

Reply:
Fixed